src/dlgPackageExporter.cpp

```diff
@@ -499,19 +498,19 @@ void dlgPackageExporter::slot_export_package()
 
     if (!imageList.isEmpty()) {
         //Create description image dir
-        QString descriptionImageDirName = QStringLiteral("%1.mudlet/description_images/").arg(tempPath);
-        QDir descriptionImageDir = QDir(descriptionImageDirName);
+        QString descriptionImagesDirName = QStringLiteral("%1.mudlet/description_images/").arg(tempPath);
```

Comment:
TBH I don't like the `<packageName>.mudlet/` sub-directory in the archive - why do we not just have a sub-directory called `metadata`  we could even put the `config.lua` in there - though that would be backward incompatible. It seems pointless to have to embed the package name inside the package archive file as that just makes more work for the reader...

:man_shrugging:

Reply:
But the reader is never doing this work, all of this is done automatically for them? And having a hidden subfolder really lessens the chance of collisions.

Reply:
This is not really part of this PR. Maybe move the discussion to a new issue?